shared/js/ui/views/whitelist.es6.js

```diff
@@ -15,19 +15,84 @@ function Whitelist (ops) {
 Whitelist.prototype = window.$.extend({},
   Parent.prototype,
   {
+    _isHiddenClass: 'is-hidden',
+    _isDisabledClass: 'is-disabled',
+    _isInvalidInputClass: 'is-invalid-input',
```

Comment:
Frankly I didn't want to declare a const for these classes in each function that uses them, and of course I don't want to just type the hardcoded string everytime - you know, typos...

I thought an external module would be a huge overkill and finally settled on this.